packages/server/src/user/user.resolver.ts

```diff
@@ -29,6 +30,14 @@ export class UserResolver {
     return this.userService.findUserByIdIfPermissionGiven(id, projectId);
   }
 
+  // get authenticated user
+  @Query(() => UserModel)
+  @UseGuards(AuthGuard)
+  async me(@UserId() userId: string): Promise<UserModel> {
+    this.logger.log(`Users of projectId: ${userId} found updated`);
```

Comment:
This log message doesn't make any sense, maybe something like this
```suggestion
    this.logger.log(`Called "me" with userId: ${userId}`);
```